docs/reference/modules/ui.md

```diff
@@ -0,0 +1,221 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/ui`
+
+ The `@apostrophecms/ui` module provides the components and mixins for creating the admin UI. It also provides the UI styling with some built-in theming capabilities that can be leveraged to create a custom admin experience.
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| `widgetMargin` | String | CSS margin value for widget spacing, default is `'20px 0'` |
```

Comment:
It should be mentioned that this applies only when logged in with editing privileges.

Reply:
Changed and better explanation added.